napari/util/io.py

```diff
@@ -1,27 +1,88 @@
+import os
+
 import numpy as np
 from skimage import io
-from glob import glob
+
+dask_available = True
+try:
+    from dask import array as da
+except ImportError:
+    dask_available = False
 
 
-def read(filenames):
-    """ Read image files and return an array.
+def imread(filenames, *, use_dask=None, stack=True):
+    """Read image files and return an array.
 
     If multiple images are selected, they are stacked along the 0th axis.
 
     Parameters
     -------
     filenames : list
         List of filenames to be opened
+    use_dask : bool
+        Whether to use dask to create a lazy array, rather than NumPy.
+        Default is None, which is interpreted as "use if available". If set
+        to True and dask is not installed, this function
+    stack : bool
+        Whether to stack the images in multiple files into a single array. If
+        False, a list of arrays will be returned.
 
     Returns
     -------
     image : array
         Array of images
     """
+    if dask_available and use_dask is None:
+        use_dask = True
+    if not dask_available and use_dask:
+        raise ValueError('Dask array requested but dask is not installed.')
     images = [io.imread(filename) for filename in filenames]
```

Comment:
Isn't this just reading the images directly in a non-lazy way? In which case I don't understand what value dask is providing. For the use dask case don't we want to use something like dask-image

I'd like to be able to do the following on the command line
```
napari my_dir_of_images/*.png -D
```
and have that use `data = dask_image.imread(my_dir_of_images/*.png)` which I know works from a script and is super cool because those pngs don't need to be all the same size